service/src/test/java/com/sedliarov/learningtable/utils/GroupFixture.java

```diff
@@ -47,49 +45,49 @@ public Group createEntityWithName(String name) {
   /**
    * Method for create Group entity and set teacher.
    */
-  public Group createEntityWithTeacher(Teacher teacher) {
+  public Group createEntityWithTeacher(UUID teacher) {
     Group group = createEntity();
-    group.setTeacher(teacher);
+    group.setTeacherId(teacher);
     return group;
   }
 
   /**
    * Method for create Group entity and set students.
    */
-  public Group createEntityWithStudents(Set<Student> students) {
+  public Group createEntityWithStudents(Set<UUID> students) {
```

Comment:
And the same for other
```suggestion
  public Group createEntityWithStudents(Set<UUID> studentIds) {
```

Reply:
Done